service/api_task.go

```diff
@@ -352,6 +352,28 @@ func assignNextAvailableTask(taskQueue *model.TaskQueue, taskQueueService model.
 			queueItem = taskQueue.FindNextTask(spec)
 		}
 		if queueItem == nil {
+			if d.PlannerSettings.Version == evergreen.PlannerVersionRevised {
+				grip.Debug(message.Fields{
```

Comment:
you can do, grip.DebugWhen(<conditional>, message.Fields) for the same impact with less indentation if you want.